plugins/modules/zos_job_output.py

```diff
@@ -132,6 +132,10 @@
         Time, local to the target system, when the job was created.
       type: str
       sample: "14:15:00"
+    execution_time:
+      description: The time the JCL executed for.
```

Comment:
I think we can be a bit more verbose about this option, pending to check how this behaves with long running jobs.
```suggestion
      description: Total duration time of the job execution, if it has finished. If the job is still running, it represents the time elapsed from the job execution start and current time.
```